modules/monitoring-configuring-metrics-collection-profiles.adoc

```diff
@@ -0,0 +1,35 @@
+// Module included in the following assemblies:
+//
+// * monitoring/configuring-the-monitoring-stack.adoc
+
+:_content-type: CONCEPT
+[id="configuring-metrics-collection-profiles_{context}"]
+= Configuring metrics collection profiles
+ 
+[IMPORTANT]
+====
+[subs="attributes+"]
+Using a metrics collection profile is a Technology Preview feature only. Technology Preview features are not supported with Red Hat production service level agreements (SLAs) and might not be functionally complete.
+Red Hat does not recommend using them in production. 
+These features provide early access to upcoming product features, enabling customers to test functionality and provide feedback during the development process.
+
+For more information about the support scope of Red Hat Technology Preview features, see link:https://access.redhat.com/support/offerings/techpreview[https://access.redhat.com/support/offerings/techpreview].
+====
+
+By default, Prometheus collects all metrics exposed by the default metrics targets in {product-title} monitoring components. 
+However, you might want Prometheus to collect fewer metrics for a cluster in certain scenarios:
+
+* If cluster administrators require only alert and console metrics, and they do not require other metrics to be available.
+* If a cluster increases in size, and the increased size of the default metrics data being collected now requires increased CPU and memory resources. 
```

Comment:
A bigger cluster will always request more CPU and memory, but feel free to ignore this suggestion if you feel like it doesn't contribute 
```suggestion
* If a cluster increases in size, and the increased size of the default metrics data being collected now requires an unreasonable increase of CPU and memory resources. 
```

Reply:
Revised to say `now requires a significant increase in CPU and memory resources`